pkg/services/chgm/mock/interfaces.go

```diff
@@ -3,257 +3,3 @@
 
 // Package mock is a generated GoMock package.
 package mock
```

Comment:
Can this file be removed completely or do we need it for something? 
We might need to change the makefile to keep it from being generated again.

Reply:
Good catch! those are just old leftover files that I didn't notice.